packages/expo-cli/src/commands/fetch/android.ts

```diff
@@ -31,7 +32,8 @@ export async function fetchAndroidKeystoreAsync(projectDir: string): Promise<voi
   await maybeRenameExistingFile(projectDir, keystoreFilename);
   const backupKeystoreOutputPath = path.resolve(projectDir, keystoreFilename);
 
-  const view = new DownloadKeystore(ctx.manifest.slug);
+  invariant(ctx.manifest.slug, 'app.json slug field must be set');
```

Comment:
in current expo-cli versions we set the slug automatically if it's not provided in app.json, so it would certainly be weird to get into this state! good to have that invariant

Reply:
were we encountering cases where slug was not set?

Reply:
i changed the type of `manifest` from `any` to `ExpoConfig`, and `slug` could be undefined in this case - so i added invariant checks to get rid of the errors